### PR TITLE
Features/work item refactor

### PR DESCRIPTION
**Pull request by davidey**
Extracted the styles from the WorkItem component.

@collingo @daaain all yours!

### CODE REVIEW

File: src/app/components/work-item-styles/index.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+import React from 'react';
+import get from 'lodash/object/get';
+
+import classnames from 'classnames';
+
+const WorkItemStyles = React.createClass({
+  render() {
+    console.log(this.props)
```
Comment (collingo) [inline src/app/components/work-item-styles/index.js]: Oops, need to kill this fella :)

File: src/app/components/work-item/index.js [hunk 2]
```diff
@@ -7,16 +7,17 @@ import Flux from '../../flux';
 
 import Rimage from '../rimage';
 import ImageHover from '../image-hover';
+import WorkItemStyles from '../work-item-styles';
+import classnames from 'classnames';
 
 const WorkItem = React.createClass({
   render() {
+    console.log(this.props)
```
Comment (collingo) [inline src/app/components/work-item/index.js]: And this guy

**Comment by collingo** [comment]
Just a couple of stray console logs otherwise looks like a nice addition and completely aligns with the spirit of composition using React :+1:

**Comment by daaain** [comment]
Nice, so this is how you do theming in React :)

:shipit:

**Pull request merged**